### PR TITLE
Fix typo in AMSI test

### PR DESCRIPTION
**Pull request by iSazonov**
## PR Summary

Now CI-Windows fail after #8546 : correct typo in cmdlet name in AMSI test.

## PR Checklist

- [x] [PR has a meaningful title](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
    - Use the present tense and imperative mood when describing your changes
- [x] [Summarized changes](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [x] [Change is not breaking](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#making-breaking-changes)
- [x] [Make sure all `.h`, `.cpp`, `.cs`, `.ps1` and `.psm1` files have the correct copyright header](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [x] This PR is ready to merge and is not [Work in Progress](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---work-in-progress).
    - If the PR is work in progress, please add the prefix `WIP:` to the beginning of the title and remove the prefix when the PR is ready.
- **User-facing changes**
    - [x] Not Applicable
    - **OR**
    - [ ] User-facing [Documentation needed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
        - [ ] Issue filed - Issue link:
- **Testing - New and feature**
    - [ ] Not Applicable or can only be tested interactively
    - **OR**
    - [x] [Make sure you've added a new test if existing tests do not effectively test the code changed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#before-submitting)
        - [x] [Add `[feature]` if the change is significant or affects feature tests](https://github.com/PowerShell/PowerShell/blob/master/docs/testing-guidelines/testing-guidelines.md#requesting-additional-tests-for-a-pr)

### CODE REVIEW

File: test/powershell/Modules/Microsoft.PowerShell.Security/AmsiInterface.Tests.ps1
Comment (vexx32) [inline test/powershell/Modules/Microsoft.PowerShell.Security/AmsiInterface.Tests.ps1]: The issue isn't the use of alias or not; the issue is that the cmdlet name was misspelled. I think removing the alias was the right move; we should keep the proper full cmdlet name.

Why did this not come up in the initial PR?

Comment (iSazonov) [reply test/powershell/Modules/Microsoft.PowerShell.Security/AmsiInterface.Tests.ps1]: Thanks! I'll try cmdlet name. In initial PR we did not run all tests with [Feature].

**Comment by iSazonov** [comment]
@SteveL-MSFT Please review to fix night tests.

**Review by SteveL-MSFT — APPROVED** [review]

File: test/powershell/Modules/Microsoft.PowerShell.Security/AmsiInterface.Tests.ps1
Comment (SteveL-MSFT) [reply test/powershell/Modules/Microsoft.PowerShell.Security/AmsiInterface.Tests.ps1]: Seems like we should be able to detect that the change is within a `Feature` test and automatically run those, let me think about this.

Comment (iSazonov) [reply test/powershell/Modules/Microsoft.PowerShell.Security/AmsiInterface.Tests.ps1]: CoreFX team uses a bot to run tests. We could do the same. 
Perhaps it makes sense to run short set of tests initially for speed. Then always run full set of tests before merge.

Comment (SteveL-MSFT) [reply test/powershell/Modules/Microsoft.PowerShell.Security/AmsiInterface.Tests.ps1]: @iSazonov that's a good idea.  Maintainers can request bot to run full set of tests after review is approved and CI tests pass.  Let me work on this.

**Pull request merged**
